src/scss/components/_listings.scss

```diff
@@ -34,6 +34,9 @@ $listing-borderRadius       : 4px;
         }
     }
 
+    &.has-no-match {
```

Comment:
Should make this helper relate to what it is, rather than what it does.  Otherwise it'll be a bit strange in other listings where `no-match` doesn't mean much.

Either that, or I'd consider making this a full on utility which (`u-stripBorder`?) which can be added potentially

Reply:
what would you call the class name? `has-child-listings`, `has-children`?
Not sure I could use a utility - I need an `is` or `has` class for the JS side of things :)

Reply:
Possibly a better naming actually would just be `has-noResults` as for a listing component that would work?

Reply:
consider it done!